examples/compiler-hydration.html

```diff
@@ -0,0 +1,53 @@
+<!-- skip-validation -->
```

Comment:
WDYT about adding an exception to `build-system/test-configs/config.js` instead, that way you don't need to modify the validation code much?

Reply:
https://github.com/ampproject/amphtml/blob/main/build-system/test-configs/config.js#L183

Reply:
Potato potato

Reply:
Making the change now

Reply:
You say potato potato I say potato potato